### PR TITLE
Set URL to gator and remove File menu

### PR DESCRIPTION
**Pull request by fcollonval**

### CODE REVIEW

**Review by jtpio — APPROVED** [review]

**Comment by jtpio** [comment]
Thanks!

The file menu and top area were mostly used as placeholder for now. So we can drop them if there is no use at the moment :+1:

**Comment by jtpio** [comment]
> The file menu and top area were mostly used as placeholder for now. So we can drop them if there is no use at the moment

But we'll probably want to bring them back at some point when new features are added. Also having a Help menu with the version number is quite useful.

File: packages/navigator/package.json [hunk 1]
```diff
@@ -44,6 +43,7 @@
     "react-dom": "~16.9.0"
   },
   "devDependencies": {
+    "@jupyterlab/buildutils": "^2.0.0",
```
Comment (jtpio) [inline packages/navigator/package.json:46]: Looks like we might be able to drop the dependency if it's not used?

https://github.com/mamba-org/gator/blob/bea611ea5d508b9da8e240cd736ae59e3d24fa75/packages/navigator/webpack.config.js#L6

**Comment by fcollonval** [comment]
Testing binder on pr action /on

File: packages/navigator/package.json (hunk 1 above)
Comment (fcollonval) [reply packages/navigator/package.json:46]: Thanks for catching that. I'll clean the code.

**Pull request merged**